### DISCUSSION TITLE
WG Server->Client port forwarding for qbittorrent

### DISCUSSION THREAD

**Discussion by MMrrTT** [wg-easy/wg-easy#458] [Q&A]
Hello.
I have installed VPN server on my VPS with this wonderful script. As I understand it made port frowarding to listen to WG port. But I use qbittorrent through VPN and as I understand for its normal use I should forward its port too, but from my VPS's WAN address to my VPN client's LAN address. The case is complicated by the fact that this is a dockered app, I don't know how to do it.
Besides I have multiple users that use my VPN server at a time. They are working fine and even if someone would need the same port I use (which is fairly rare situalion) I agree that it won't work for them :)
For example I need to forward 9999 port from external VPS's IP (let's say 45.45.45.45) to VPN client's IP (let's say 10.10.10.10). What should I do?

**Comment by pheiduck** [comment]
@MMrrTT 
Please provide more Infomation.
Which Version do you use latest, 9,8,7...

**Reply by k2helix** [reply]
I'm trying to achieve this exact configuration. I'm using the latest version. 

I also created an issue that got converted into a discussion that can be found here: https://github.com/wg-easy/wg-easy/discussions/1189
